### PR TITLE
🔨 Check public key in `userId` rather than cosmos address

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: src/routes/users/registerLogin.js [hunk 1]
```diff
@@ -162,7 +162,7 @@ router.post(
           payload.displayName = user;
           payload.email = email;
           payload.isEmailVerified = false;
-          platformUserId = cosmosWallet;
+          platformUserId = publicKey;
```
Comment (nwingt) [inline src/routes/users/registerLogin.js:175]: the above `cosmosWallet` should be generated by `publicKey` instead?

Comment (WeiJun0827) [reply src/routes/users/registerLogin.js:175]: I see. So the frontend no longer provides `cosmosWallet`.

Comment (nwingt) [reply src/routes/users/registerLogin.js:175]: Is it also needed to set `likeWallet` as well?

Comment (WeiJun0827) [reply src/routes/users/registerLogin.js:175]: Hmm, right, so that we have a `likeWallet` field in DB.

**Review by williamchong — COMMENTED** [review]
Should we change `checkCosmosSignPayload` to check for public key instead of cosmosWallet in memo?

**Comment by williamchong** [comment]
Due to pubkey not migrate-able, we would be doing likeWallet as address first
https://github.com/likecoin/likecoin-api-public/pull/281

**Pull request closed without merging**